packages/frontend/src/components/Game/GameBoard/index.test.jsx

```diff
@@ -1,5 +1,80 @@
+import { setupTests } from 'helpers/tests';
 import GameBoard from '.';
+import { mockPhotos } from '__mocks__/api/mockPhotos';
+import { createTilesFromPhotos } from 'helpers/createTilesFromPhotos';
+import { baseState } from 'contexts';
+import { produce } from 'immer';
+import { act } from '@testing-library/react';
+import { byTestId, byAltText } from 'testing-library-selector';
+
+beforeEach(() => {
+  jest.useFakeTimers('legacy');
+});
+
+afterEach(() => {
+  jest.useRealTimers();
+});
+
+jest.spyOn(global, 'setTimeout');
```

Comment:
Is this needed? There once existed some logic that looked if `setTimeout` was called, but I don't see that logic anymore.

Reply:
The spyOn is not needed, good catch. 

I also realized that an advance timers call is not needed in the first test, so I removed that as well.